web-server/mime.types

```diff
@@ -0,0 +1,4 @@
+types {
+  text/html                             html htm shtml;
+  text/css                              css;
```

Comment:
Do we need to configure mime types for other file types too? (we have svg/png/ico I think)